tests/console/yast2_vnc.pm

```diff
@@ -27,17 +28,24 @@ sub configure_remote_admin {
     send_key $remote_admin{allow_remote_admin_with_session};
     assert_screen 'yast2_vnc_allow_remote_admin_with_session';
     # Firewall Settings
-    if (check_screen 'yast2_vnc_firewall_port_closed') {
-        send_key $firewall_settings{open_port};
+    # bsc#1207390 skip open firewall ports for tumbleweed
+    if ((check_screen 'yast2_vnc_no_network_interface') && is_tumbleweed) {
```

Comment:
the cheapest operation first:
`is_tumbleweed && check_screen 'yast2_vnc_no_network_interface')`